dev/index.html

```diff
@@ -124,6 +124,11 @@
           document
             .getElementById("heatmapHotmap")
             .setHeatmapData(xDomain, yDomain, data);
+
+          // await document.getElementById("heatmapHotmap").updateComplete.then(() => {
```

Comment:
Delete this line if the are not going to be in use

Reply:
Done